.github/workflows/nodejs_build.yml

```diff
@@ -15,10 +15,12 @@ jobs:
     steps:
       - uses: actions/checkout@v3
       - name: Use Node.js ${{ matrix.node-version }}
+        if: always()
```

Comment:
This needs to be added to the lint script that you added.

Reply:
So in the json file then?

Reply:
No. Still in this file, but on the format script that runs inside here

Reply:
I thought it was the git actions error that you talked about today is the error happening with npm.

Reply:
What do you mean?

Reply:
What we want to do is run the lint command regardless if the build command fails. If you check the GitHub action logs you can see that's not happening

Reply:
Yes I understand that.
`name: Node.js CI

on:
  pull_request:
    branches: [main]

jobs:
  build:
    runs-on: ubuntu-latest

    strategy:
      matrix:
        node-version: [20.x]

    steps:
      - uses: actions/checkout@v3
      - name: Use Node.js ${{ matrix.node-version }}
        uses: actions/setup-node@v3
        with:
          node-version: ${{ matrix.node-version }}
      - run: npm ci
      - run: npm run build
        env:
          CI: true
        if: always()
      - run: npm run format:check
`
So adding it above the linting check step should make it so that it always runs correct?

Reply:
I think so. I'm not sure if we need to nest the run command under some other keys or not though